js/app.js

```diff
@@ -120,6 +168,14 @@
 	*/
 
 	// implement function here
+	const removeFirstItem = (list) => {
+		if (!Array.isArray(list) || list.length === 0) {
+			return;
+		}
+
+		return list.splice(1, list.length);
```

Comment:
I believe that we can work on this one a bit to make sure that `list.length` isn't overwhelming the splice.